examples/ITK_Example17_MONAIWithPreregistration.ipynb

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    from monai.apps import MedNISTDataset
</pre><p>Could a cell at the top please be added to install dependencies via pip? This improves reproduciblity and portability. We want the CI tests to pass.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKElastix/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Until we get to that point, we should install the dependencies here: https://github.com/InsightSoftwareConsortium/ITKElastix/blob/main/.binder/requirements.txt